extensions-contrib/opentelemetry-extensions/pom.xml

```diff
@@ -85,6 +85,11 @@
             <artifactId>junit</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>nl.jqno.equalsverifier</groupId>
+            <artifactId>equalsverifier</artifactId>
+            <scope>test</scope>
+        </dependency>
```

Comment:
The Equals tests are required to meet the code coverage threshold. (More details in the comment [below](https://github.com/confluentinc/druid/pull/129#discussion_r1105703402))

Given there are 10 fields to check, I think it would be better to use this library as it covers all the negative  branch cases too.

Reply:
ok, I wonder why we don't need it for metrics input format. It looks like we already use this dependency elsewhere in druid so I'm ok using that here.